debian/control

```diff
@@ -5,7 +5,7 @@ Maintainer: daos-stack <daos@daos.groups.io>
 Build-Depends: debhelper (>= 10),
                dh-python,
                libcmocka-dev,
-               libfabric-dev (>= 1.15.1-1),
+               libfabric-dev (= 1.15.1-1),
```

Comment:
We should **never** use an `=` pin.  It prevents us from releasing a bug-fix release (i.e. 1.15.2) in the future without also going through the whole (big and heavy) daos release process.

Reply:
What should we use then? <= 1.16?